### ISSUE TITLE
Hand detection

### ISSUE THREAD

**Issue by JVpltr** [floe/backscrub#85]
Hi, 
First thanks for all your work, 
After testing all the model given here i realize that none of then is really good for detecting hand. Do you know any model which is good at detecting people and hands? 

Thanks

**Comment by floe** [comment]
I think this here would fit the bill: https://google.github.io/mediapipe/solutions/hands.html

**Comment by JVpltr** [comment]
Thanks you for your answer,  but if I use 2 differents model it risk to have "hole" in the mask where arms are  no?

**Comment by floe** [comment]
Oh, now I know what you mean... I guess that would involve improving the training set and re-training the CNN. Did you try any of the other models, in some scenarios they can give better results?

**Comment by JVpltr** [comment]
Yeah I tried to retrained 2 or 3 models but I'm not a not really a pro in ia so my training were certainly not the best . And I didn't find good data set with hand involved.
